keep-ui/shared/ui/Select/Select.tsx

```diff
@@ -0,0 +1,131 @@
+"use client";
```

Comment:
how isMulti is used here with multiselect? isn't it only the style?

Reply:
`isMulti` used for style type annotations here, yes. the prop `isMulti` passed down to react-select